package/homebrew.spec

```diff
@@ -0,0 +1,59 @@
+Summary:   Homebrew package manager
+Name:      homebrew
+Version:   4.1.5
```

Comment:
Need to figure out how to make this dynamic

Reply:
Could be like an `.in` file. Just make this a placeholder and `sed` it in as a part of the build process: https://github.com/Homebrew/ci-orchestrator/blob/d44fac1b126b8562fa3e8574bbc5dfd07b043de5/.github/workflows/main.yml#L89-L101

Reply:
It should be, check the actions changes. Just need to fix that to create a tarbal of the current checkout before making the rpm.

Reply:
A missed that bit. Maybe worth making this a placeholder value that appears obviously wrong if built.

Reply:
Done, I only realised later that my original idea for tarbals was too limited. The tarring in the action is better.